mmengine/hooks/checkpoint_hook.py

```diff
@@ -304,6 +325,64 @@ def after_val_epoch(self, runner, metrics):
 
         self._save_best_checkpoint(runner, metrics)
 
+    def after_train(self, runner) -> None:
+        """Publish the checkpoint after train epoch.
+
+        Args:
+            runner (Runner): The runner of the training process.
+        """
+        if not self.published_keys:
+            return
+
+        if self.save_last:
+            last_ckpt = runner.message_hub.get_info('last_ckpt')
+            if not last_ckpt:
+                assert ('Did not find last_checkpoint to be resumed.')
```

Comment:
What will happen if `last_ckpt` is not in `message_hub` when we call `runner.message_hub.get_info('last_ckpt')`
```suggestion
            assert last_ckpt ('Did not find last_checkpoint to be resumed.')
```